MAUI/DataGrid/Styling.md

```diff
@@ -50,143 +54,167 @@ You can write custom style for the properties in SfDataGrid.DefaultStyle class u
+        <Style TargetType="syncfusion:DataGridCell">
+            <Setter Property="Background" Value="Bisque"/>
+            <Setter Property="TextColor" Value="Red"/>
+            <Setter Property="FontAttributes" Value="Italic"/>
+            <Setter Property="FontSize" Value="14"/>
+            <Setter Property="FontFamily" Value="TimesNewRoman"/>
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
 {% endhighlight %}
 {% endtabs %}
 
 ![Record cell styling in .NET MAUI DataGrid](Images/styling/maui-datagrid-recordcell-style.png)
 
 ### Styling Record row
-The record row can be customized by writing style of TargetType DataGridRow. 
+The record row can be customized by writing style of TargetType [DataGridRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridRow.html). 
 
 {% tabs %}
 {% highlight xaml tabtitle="MainPage.xaml"%}
-<Style TargetType="syncfusion:DataGridRow">
-    <Setter Property="Background" Value="Bisque"/>
-</Style>
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Resources>
+        <Style TargetType="syncfusion:DataGridRow">
+            <Setter Property="Background" Value="Bisque"/>
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
 {% endhighlight %}
 {% endtabs %}
 
 ![Record row styling in .NET MAUI DataGrid](Images/styling/maui-datagrid-recordrow-style.png)
 
 ### Styling Header cell
-The header cells can be customized by writing style of TargetType DataGridHeaderCell
+The header cells can be customized by writing style of TargetType [DataGridHeaderCell] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridHeaderCell.html).
 
 {% tabs %}
 {% highlight xaml tabtitle="MainPage.xaml"%}
-<Style TargetType="syncfusion:DataGridHeaderCell">
-    <Setter Property="Background" Value="Bisque"/>
-    <Setter Property="TextColor" Value="Red"/>
-    <Setter Property="FontAttributes" Value="Bold"/>
-    <Setter Property="FontSize" Value="14"/>
-    <Setter Property="FontFamily" Value="TimesNewRoman"/>
-</Style>
+<ContentPage xmlns:syncfusion="http://schemas.syncfusion.com/maui">
+    <ContentPage.Resources>
+        <Style TargetType="syncfusion:DataGridHeaderCell">
+            <Setter Property="Background" Value="Bisque"/>
+            <Setter Property="TextColor" Value="Red"/>
+            <Setter Property="FontAttributes" Value="Bold"/>
+            <Setter Property="FontSize" Value="14"/>
+            <Setter Property="FontFamily" Value="TimesNewRoman"/>
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
 {% endhighlight %}
 {% endtabs %}
 
 ![Header cell styling in .NET MAUI DataGrid](Images/styling/maui-datagrid-headercell-style.png)
 
 ### Styling Header row
-The header row can be customized by writing style of TargetType DataGridHeaderRow.
+The header row can be customized by writing style of TargetType [DataGridHeaderRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridHeaderRow.html).
```

Comment:
The header row can be customized by the writing style of TargetType [DataGridHeaderRow] (https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridHeaderRow.html).